Playbooks/AS-Create-Opsgenie-Incident/README.md

```diff
@@ -0,0 +1,133 @@
+# AS-Create-Opsgenie-Incident
+
+Author: Accelerynt
+
+For any technical questions, please contact info@accelerynt.com  
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FAS-Create-Opsgenie-Incident%2Fazuredeploy.json)
+[![Deploy to Azure Gov](https://aka.ms/deploytoazuregovbutton)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FAS-Create-Opsgenie-Incident%2Fazuredeploy.json)       
+
+This playbook is intended to be run from a Microsoft Sentinel Incident. It will create either an incident or an alert in Opsgenie with the information from a Microsoft Sentinel incident.
+

```

Comment:
Attach a screenshot of playbook here.

Reply:
@rahul0216 are you requesting I add a screen shot of a successful run or of the playbook in the edit mode so the steps are visible? Are you wanting this in the chat for your reference or are you wanting me to add this to the README file?

Reply:
@AcceleryntSecurityDev Sorry for the confusion.
Both actually,
1. Add screenshot of playbook in edit mode to README file like this (https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/QualysVM/Playbooks/QualysVMPlaybooks/QualysVM-GetAssetDetails/readme.md)
2. Add screenshot of successful run here in comment, to verify the successful run of playbook